priv/.exavier.exs

```diff
@@ -0,0 +1,6 @@
+%{
```

Comment:
If this is just a template, I would include all possible options, but have all of them commented out and include documentation about this configuration inline (or at least links to the documentation in the comments). That way it's clear that this is optional config and that it can work without any configuration.

Reply:
Done! ✅